app/default_overrides_test.go

```diff
@@ -80,11 +80,11 @@ func TestDefaultConsensusConfig(t *testing.T) {
 			Size:                  tmcfg.DefaultMempoolConfig().Size,
 			WalPath:               tmcfg.DefaultMempoolConfig().WalPath,
 
-			// overrides
-			MaxTxBytes:   7_897_088,
-			MaxTxsBytes:  39_485_440,
+			// Overrides
+			TTLNumBlocks: 12,
 			TTLDuration:  75 * time.Second,
-			TTLNumBlocks: 5,
+			MaxTxBytes:   8 * mebibyte,
```

Comment:
I don't see the harm in keeping this at 8, but it might be less confusing to keep it at 2MiB since that will be the hard coded limit

Reply:
yes! let's change this and I'll approve @rootulp

Reply:
The main problem with dropping this to 2MB is when we decide to increase the tx size limit, we will also need to increase it here (and tell validators to modify their configs else large transactions won't get in). I would opt to leave it 8MB.

Once again we still have the issue of not knowing whether a config has been modified by a user or not to know whether we can override it if we decide to change it

Reply:
> I don't see the harm in keeping this at 8, but it might be less confusing to keep it at 2MiB since that will be the hard coded limit

The current default is ~7.5 MiB so you're proposing a large decrease. Do you mind re-reading the PR description? 

 cc: @evan-forbes @ninabarbakadze

Reply:
To clarify: I think it's confusing to keep the existing ~7.5 MiB. 

If the proposal is to decrease `MaxTxBytes` to 2 MiB, I thought that was actually a solution to https://github.com/celestiaorg/celestia-app/issues/3686 and I asked about it [here](https://github.com/celestiaorg/celestia-app/issues/3686#issuecomment-2393738597). AFAICT there wasn't alignment on setting `MaxTxBytes` to 2 MiB.

Reply:
I do see @cmwaters point but I still think it's counter intuitive to set the default to ~7.5 MiB when we inevitably reject txs that are > 2MiB in CheckTx

Reply:
Resolved via https://github.com/celestiaorg/celestia-app/pull/3961/commits/e7f9a6c000f1234369c7f6366ded2d7271587e40